e2e/app/page/workspace-edit-page.ts

```diff
@@ -271,10 +271,7 @@ export default class WorkspaceEditPage extends WorkspaceBase {
     // Wait for Workspace name text-field, Billing Account Select.
     await Promise.all([this.getWorkspaceNameTextbox().waitForXPath(), select.waitForXPath()]);
     // Build Workspace page is used for Duplicate and Create. Wait for Create or Duplicate button.
-    await Promise.race([
-      this.getCreateWorkspaceButton().waitForXPath({}),
-      this.getDuplicateWorkspaceButton().waitForXPath({})
-    ]);
+    await this.getCancelButton().waitForXPath();
```

Comment:
Checking CANCEL button instead CREATE or DUPLICATE button to avoid non-fatal error thrown:

```
ERROR: [4/23/2021, 20:36:59] - waitForXpath('(//button | //*[@role="button"])[text()="Create Workspace" or @aria-label="Create Workspace" or @placeholder="Create Workspace" or @value="Create Workspace"]') failed

ERROR: [4/23/2021, 20:36:59] - Protocol error (Runtime.callFunctionOn): Target closed.
ERROR: [4/23/2021, 20:36:59] - Error: Protocol error (Runtime.callFunctionOn): Target closed.
    at /home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/Connection.js:208:63
    at new Promise (<anonymous>)
    at CDPSession.send (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/Connection.js:207:16)
    at ExecutionContext._evaluateInternal (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:200:50)
    at ExecutionContext.evaluateHandle (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:151:21)
    at WaitTask.rerun (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:528:37)
```